### ISSUE TITLE
All Wizards of the Coast Materials Can't Load With Same Core Sets

### ISSUE THREAD

**Issue by PsychoCircus** [BahamutDragon/pcgen#193]
There are two separate sets of core books for the WOTC materials, the Dungeons & Dragons Core Books, and the individual books (Player's Handbook, Dungeon Master's Guide, and Monster Manual I).  Most of the other sources (Forgotten Realms stuff, etc) will load with one or the other, but some materials will only load with one set or the other.  This means if you want to load ALL the WOTC material, you have to load 90% of it with one set of core books, then unload those core books and load the others so you can load the remaining material.  Either way, this results in errors and red sources because you can't load everything from one set, and you can't load both sets of core materials at once.

**Comment by BahamutDragon** [comment]
Mate,

Pushed patch into both branches. Test and see if that corrected everything.

**Comment by PsychoCircus** [comment]
I'm still getting red sources and having to load from two different sets of core books

**Comment by BahamutDragon** [comment]
Only Savage Species is red for me mate. Something I will push in a fix tonight. Are you trying to load other sources that are not part of the repository? Can you give me a couple of books you cannot load?

**Comment by BahamutDragon** [comment]
@PsychoCircus waiting on a reply mate.

**Comment by PsychoCircus** [comment]
Sorry about that.  I've had a very hectic past week, and haven't had a chance to actually sit down with it again.  I'll load it up here in a bit and let you know what I can do.  Thank you again

**Comment by BahamutDragon** [comment]
Hey Mate, sitting here with this song playing in my head...
https://www.youtube.com/watch?v=0Wi8Fv0AJA4

Any updates or time frame for follow up? :-)

**Comment by PsychoCircus** [comment]
Sorry about that.  Things are still really crazy for me at the moment.  I just tried again and everything is still red, in fact if I unload it, I can't reload it normally.  But, I took a look at the debug log and I'm seeing quite a few problems.   A lot of them are unconstructed references, illegal tokens and that sort of thing, but I've also got a problem with multiple skills and sources with the same name.  So I'm thinking that's a problem on my end, at least a lot of it is

So, that in mind, I think I'm going to uninstall PCGen completely, download a fresh copy, install it, and do the same for your repository, and install it and try that.  And let everything install default locations instead of me changing it.  That way at least I'll have a better idea of what's going on.

Oh, about the other questions in the other closed thread, I had no preference for what sort of dragon was worked on first, though the shadow dragon was the one I was working with.  And I'd love to help in any way I can.  I haven't done anything like this in quite some time though, so I'm a bit rusty, but picking things back up should be easy enough.  My biggest challenge is being able to sit down and do it.  But if at all possible, I'm happy to help as much as I can when I can.  Thanks for all your help so far, man, and more in the future as I'm going to need it I'm sure!

**Comment by BahamutDragon** [comment]
@PsychoCircus a closed issue typically means fixed, finished or otherwise implemented, unless commented otherwise. 

All the dragons for the other thread are implemented. The breath weapons do not calculate yet, but as far as implementation, everything is now present that was part of the original issue.

**Comment by BahamutDragon** [comment]
Closing this as resolved. If there is still an issue, comment with specific sources. Best,
